src/CMakeLists.txt

```diff
@@ -73,6 +73,17 @@ if(F3D_MODULE_ASSIMP)
   endif()
 endif()
 
+if(F3D_MODULE_ALEMBIC)
+  set(MODULES_SPECIFIC_SOURCE_FILES ${MODULES_SPECIFIC_SOURCE_FILES} "${CMAKE_CURRENT_SOURCE_DIR}/vtkF3DAlembicImporter.cxx")
+  set(MODULES_SPECIFIC_LIBRARIES ${MODULES_SPECIFIC_LIBRARIES} Alembic::Alembic)
+
+  # Note: Alembic CMake module file does not declare Debug target properties, just map it to Release properties
```

Comment:
Is it copied from `Assimp` integration?  
It's probably not needed, it's mostly a hack to solve an issue in Assimp CMake files.

Reply:
I have remove the redundant cmake commands.